TripCheckins/Trip list/UserDefaults+LocalItemsStorage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UserDefaults+LocalItemsStorage.swift
+//  TripCheckins
+//
+//  Created by Nataliya  on 8/27/17.
+//  Copyright © 2017 Nataliya Patsovska. All rights reserved.
+//
+
+import Foundation
+
+let tripsKey = "trips"
+
+extension UserDefaults: LocalItemsStorage {
+    
+    func storedTrips() -> [Trip] {
```

Comment:
naming is a little confusing - storeTrip vs storedTrips. Maybe something more explicit about retrieving/fetching/getting trips.